### PR TITLE
dcos-diagnostics: bump to 84f0d569

### PR DESCRIPTION
**Pull request by jdef**
## High-level description

* support for optional HTTP endpoints

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-5031](https://jira.mesosphere.com/browse/DCOS_OSS-5031) DC/OS Diagnostics: HTTP endpoint targets can be marked optional.

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS-51618](https://jira.mesosphere.com/browse/DCOS-51618) DC/OS Diagnostics should include output of DSS /diagnostics endpoint.

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: N/A (changes to configuration + related tests will come later)
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): https://github.com/dcos/dcos-diagnostics/pull/101
  - [x] Test Results: see above PR
  - [x] Code Coverage see above PR
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by jdef** [comment]
@mesosphere-mergebot label ready for review

**Comment by jdef** [comment]
@janisz @gpaul PTAL

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes  Can you review, please?

File: CHANGES.md [hunk 1]
```diff
@@ -137,6 +137,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * DC/OS UI X-Frame-Options default value has been changed from `SAMEORIGIN` to `DENY`. This is now configurable using the `adminrouter_x_frame_options` configuration value (DCOS-49594)
 
+* HTTP endpoint targets of DC/OS Diagnostics can be marked optional (DCOS_OSS-5031)
```
Comment (makkes) [inline CHANGES.md]: Is there documentation for this anywhere?

Comment (jdef) [reply CHANGES.md]: No, I didn't see any existing documentation set to update for dcos-diagnostics. If you can link me to an existing doc I'll be happy to update it.

Comment (makkes) [reply CHANGES.md]: @janisz is there anything like it?

**Comment by jdef** [comment]
@mesosphere-mergebot bump-ee

**Comment by jdef** [comment]
ugh, is there a way to avoid dismissing reviews when rebasing to the latest master?

**Comment by gpaul** [comment]
> ugh, is there a way to avoid dismissing reviews when rebasing to the latest master?

nope :-/

**Comment by jdef** [comment]
@janisz mind re-approving? I force-pushed to rebase, not realizing that it was not needed - and mergebot dismissed your review

File: CHANGES.md (hunk 1 above)
Comment (janisz) [reply CHANGES.md]: Nope, it's internal config. But yeah we should document it in dcos-diagnostics repo

**Pull request merged**